service/worker/scheduler/workflow.go

```diff
@@ -175,6 +175,7 @@ type (
 		ReuseTimer                        bool                     // Whether to reuse timer. Used for workflow compatibility.
 		NextTimeCacheV2Size               int                      // Size of next time cache (v2)
 		SpecFieldLengthLimit              int                      // item limit per spec field on the ScheduleInfo memo
+		SpecVersion                       SpecVersion              // version for spec logic
```

Comment:
Just curious, why not just a simple bool?

Reply:
Basically the same reason for `Version`: if we added a bool for each fix, the struct would get bigger and bigger. This way we can just increment a number.